ui/app/helpers/date-from-now.js

```diff
@@ -0,0 +1,8 @@
+import { helper } from '@ember/component/helper';
+import { distanceInWordsToNow } from 'date-fns';
```

Comment:
So I dug into why this happens - it looks like by default date-fns exports everything as one giant object. I wonder if there's some webpack config we can do to get the tree shaking. (not necessary for this PR though)